src/System Application/App/Azure Storage Services Authorization/src/ReadySAS/StorServAuthReadySAS.Codeunit.al

```diff
@@ -29,28 +29,36 @@ codeunit 9088 "Stor. Serv. Auth. Ready SAS" implements "Storage Service Authoriz
 
         if QueryText <> '' then
             QueryText += '&';
-        QueryText += GetSharedAccessSignature();
+        QueryText += GetSharedAccessSignature().Unwrap();
         UriBuilder.SetQuery(QueryText);
 
         UriBuilder.GetUri(Uri);
 
         HttpRequestMessage.SetRequestUri(Uri.GetAbsoluteUri());
```

Comment:
Swap `SetRequestUri` for `SetSecretRequestUri`. This prevents the retrieval of the signature if the HttpRequestMessage were to be accessible.